packages/storage/test/service.test.ts

```diff
@@ -111,7 +112,7 @@ describe('Firebase Storage > Service', () => {
       const error = testShared.assertThrows(() => {
         new Service(testShared.fakeApp, xhrIoPool, 'gs://bucket/object/');
       }, 'storage/invalid-default-bucket');
-      assert.match(error.message, /Invalid default bucket/);
+      error && assert.match(error.message, /Invalid default bucket/);
```

Comment:
`assert.match()` should always execute. `testShared.assertThrows()` returns a `FirebaseStorageError`, I don't think we need to null check error.

Reply:
Done.